tests/Integration/Behaviour/Features/Context/LanguageFeatureContext.php

```diff
@@ -52,23 +51,10 @@ public static function restoreLanguagesTablesAfterFeature(): void
 
     private static function restoreLanguagesTables(): void
     {
-        // Removing Language manually includes cleaning all related lang tables, this cleaning is handled in
-        // Language::delete in a more efficient way than relying on table restoration
-        $langIds = Db::getInstance()->executeS(sprintf('SELECT id_lang FROM %slang;', _DB_PREFIX_));
-        unset($langIds[0]);
-        foreach ($langIds as $langId) {
-            $lang = new Language($langId['id_lang']);
-            $lang->delete();
-        }
-
-        // We still restore lang table to reset increment ID
-        DatabaseDump::restoreTables(['lang', 'lang_shop']);
+        LanguageResetter::resetLanguages();
 
         // Reset default language
         Configuration::updateValue('PS_LANG_DEFAULT', 1);
```

Comment:
is there a reason for not putting this part into `LanguageResetter::resetLanguages` ?

I would have think that setting `PS_LANG_DEFAULT` to 1 in the table `ps_lang` would be part of the reset

Reply:
Most of the time we don't change the default language so it was not necessary in our use cases But I guess it could be part of it indeed